### DISCUSSION TITLE
how to fix this error for flux??

### DISCUSSION THREAD

**Discussion by rafiislambd** [lllyasviel/stable-diffusion-webui-forge#1012] [Q&A]
how to fix this error??
model flux1-schnell-nf4.safetensors
https://huggingface.co/silveroxides/flux1-nf4-weights/blob/main/flux1-schnell-nf4.safetensors

![00](https://github.com/user-attachments/assets/b650874d-3d6c-4c43-9b9a-4c46ecb72325)
![0](https://github.com/user-attachments/assets/cb07e773-e1e1-4ebf-890e-59237e1d8eed)

**Comment by ishadowx27** [comment]
If you updated an existing installation, the best and easiest course of action would be a Clean Install. The same thing happened to me even when using the exact models suggested.
Alternatively, you could try the models listed on the "[Major Update] BitsandBytes Guidelines and Flux" suggested by Illyasviel. 
Also, do not forget to apply the settings once you update since there are many settings that change. Moreover, remove ALL extensions from the Extensions folder, just in case. I can even Generate several times faster on a 1070 8GB using DPM++ 2M with SGM_Uniform since I cleaned installed it.

**Reply by ishadowx27** [reply]
Incredibly likely. I have no optimizations enabled whatsoever and everything works great.
btw, I would also disable/delete all user inserted data in the webui-user.bat file.
![aergWGergWERGesg](https://github.com/user-attachments/assets/1eab5aa2-52eb-4f6c-bafd-2a58a5e3b90a)

**Comment by DenOfEquity** [comment]
based on the link and model hash, you have the wrong model. You need the larger all-in-one file.
https://huggingface.co/silveroxides/flux1-nf4-weights/blob/main/flux1-schnell-bnb-nf4.safetensors

**Comment by fabiobtex** [comment]
I have this error whenever I change something, to temporarily resolve it I alternate between "auto" and "nf4".

**Reply by PeterJared** [reply]
It did not work for me. There re multiple post commenting the same error with different solutions but none of them work for me in a clean install.

**Reply by fabiobtex** [reply]
Are you trying high-res fix? High-res fix is beyond help

**Reply by PeterJared** [reply]
No, simply use it. I found a workaround. The problem seems to be the safe tensor I got. One from the official page and another from  another page. Both give me that error. But I installed it from Pinokio and it downloaded a new safe tensor that works both in Pinokio  installationForge and Forge from this site.
